src/lib/strings.js

```diff
@@ -0,0 +1,49 @@
+/*eslint-disable */
+
+/**
```

Comment:
why are we eslint disabling?

Reply:
my IDE was giving me errors related to the various replace regex like strings that are used throughout these scripts:
```
.replace(/\-/g, ' ')
.replace(/(\d)(?=(\d{1})+$)/g, '$1 ')
.replace(/\s[a-z]/g, upperCase)
.replace(/\s+/g, '')
.replace(/^[A-Z]/g, lowerCase);
```

Reply:
ok, we can keep the lint disabling